src/utils/url.ts

```diff
@@ -1,7 +1,7 @@
 import { ASSETS_ROOT } from '../constants';
 
-export function wrapUtm(url: string) {
-  return `${url}?utm_medium=email&utm_source=email`;
+export function wrapUtm(url?: string, wrapUtmPrefix: string = '?') {
+  return `${url}${wrapUtmPrefix}utm_medium=email&utm_source=email`;
```

Comment:
Can keep naming `prefix` for simplicity￼ 😉
```suggestion
export function wrapUtm(url?: string, prefix: string = '?') {
  return `${url}${prefix}utm_medium=email&utm_source=email`;
```

Reply:
gotcha! changed :D